testing/sdk_cmd.py

```diff
@@ -32,3 +33,12 @@ def run_cli(cmd, print_output=True):
         log.error(err)
         raise dcos.errors.DCOSException(err)
     return stdout
+
+
+def convert_output_to_list(output):
```

Comment:
Yes, I just looked at it again. Missed the fact that the output is just a string and not the lines from the output.